pkg/proto/protodep.lock

```diff
@@ -0,0 +1,31 @@
+proto_outdir = "./protodep"
+
+[[dependencies]]
+  target = "github.com/mwitkow/go-proto-validators"
+  revision = "32a686adf8b5194d3ea07d632d49b6fb344af678"
+  branch = "master"
+  path = "github.com/mwitkow/go-proto-validators"
```

Comment:
I wonder if maybe we should add versions in the path for these? Might be also valuable to add a comment about what exact version matches the `revision`s for easier maintenance.

Reply:
This file is auto-generated by protodep based of `pkg/proto/protodep.toml` where the revisions are explicit.